openedx/adg/lms/webinars/helpers.py

```diff
@@ -0,0 +1,69 @@
+"""
+Helper methods for webinars
+"""
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+
+from openedx.adg.common.lib.mandrill_client.tasks import task_send_mandrill_email
+
+
+def validate_email_list(emails):
+    """
+    Validate list of comma separated emails addreses
+
+    Arguments:
+         emails (str): string of comma separated emails
+
+    Returns:
+        error (boolean): True if error is found, False otherwise
+        emails (list): list of parsed email
+    """
+
+    emails = [item.strip() for item in emails.split(',') if item.strip()]
+
+    for email in emails:
+        try:
+            validate_email(email)
+        except ValidationError:
+            return True, emails
+
+    return False, emails
+
+
+def webinar_email_list(webinar):
+    """
+    Given a webinar, generate a list of emails for co-host, panelists, and presenter
+
+    Arguments:
+         webinar (obj): webinar object
+
+    Returns:
+        emails (list): list of all email
+    """
+
+    panelist_emails = list(webinar.panelists.exclude(email='').values_list('email', flat=True))
+    co_host_emails = list(webinar.co_hosts.exclude(email='').values_list('email', flat=True))
+    emails = panelist_emails + co_host_emails
+    if webinar.presenter:
+        emails.append(webinar.presenter.email)
+    return emails
+
+
+def _send_webinar_emails(webinar, recipient_emails, template):
+    """
```

Comment:
why do we need to indicate this function as private/for internal use (referring to the leading underscore)?